packages/devtools/src/composite.ts

```diff
@@ -9,15 +9,21 @@ import type {
   StreamCommits,
 } from '@glazed/types'
 import type { GraphQLSchema } from 'graphql'
-import cloneDeep from 'lodash-es/cloneDeep'
-import merge from 'lodash-es/merge'
+import cloneDeep from 'lodash-es/cloneDeep.js'
```

Comment:
Is the `.js` extension needed here please? I thought Node would resolve them as expected for packages?

Reply:
Ok, so if I remove the extensions here and run tests for `packages/cli`, I get errors like this:
```
    Command failed with exit code 2: glaze composite:create test/mocks/composite.schema --key=571e3539d1153fbea02465945484d4244730fa57f77f8b7818de2b843b288c9f
    (node:17607) [MODULE_NOT_FOUND] ModuleLoadError Plugin: @glazed/cli: [MODULE_NOT_FOUND] import() failed to load /Users/arturwdowiarski/src/js-glaze/packages/cli/dist/commands/composite/create.js
    module: @oclif/core@1.9.0
    task: toCached
    plugin: @glazed/cli
    root: /Users/arturwdowiarski/src/js-glaze/packages/cli
    See more details with DEBUG=*
    (Use `node --trace-warnings ...` to show where the warning was created)
     ›   Error: command composite:create not found

      24 |       const seed = stripAnsi(key.stderr.toString().split('with seed ')[1])
      25 |
    > 26 |       const create = await execa('glaze', [
         |                      ^
      27 |         'composite:create',
      28 |         'test/mocks/composite.schema',
      29 |         `--key=${seed}`,

      at makeError (node_modules/execa/lib/error.js:59:11)
      at handlePromise (node_modules/execa/index.js:119:26)
      at Object.<anonymous> (test/composites.test.ts:26:22)

```
I then do `node /Users/arturwdowiarski/src/js-glaze/packages/cli/dist/commands/composite/create.js` to see why this module cannot be imported and I get this error:
```
node:internal/process/esm_loader:94
    internalBinding('errors').triggerUncaughtException(
                              ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/arturwdowiarski/src/js-glaze/node_modules/lodash-es/cloneDeep' imported from /Users/arturwdowiarski/src/js-glaze/packages/devtools/dist/composite.js
Did you mean to import lodash-es/cloneDeep.js?
    at new NodeError (node:internal/errors:372:5)
    at finalizeResolution (node:internal/modules/esm/resolve:437:11)
    at moduleResolve (node:internal/modules/esm/resolve:1009:10)
    at defaultResolve (node:internal/modules/esm/resolve:1218:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:580:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:294:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:80:40)
    at link (node:internal/modules/esm/module_job:78:36) {
  code: 'ERR_MODULE_NOT_FOUND'
  ```
  
  That's why I added the extensions here. Am I doing something wrong?